packages/terra-consumer-nav/src/components/user-profile/UserProfile.jsx

```diff
@@ -32,10 +33,6 @@ const propTypes = {
    */
   signinUrl: PropTypes.string,
```

Comment:
can you update the comments here to clearly reflect what might render/happen if certain things are passed in?
I'm not sure we should change whats rendering based off if this is passed in (maybe just no sign in link at all if its not passed in?). I think most apps will have a static signIn and signOut url so they will always be passing them in

Reply:
yeah it could get confusing for people that didnt write the code, will comment this